schema/deploy/shipping/views.sql

```diff
@@ -1476,7 +1476,16 @@ create or replace view shipping.hcov19_presence_absence_result_v1 as
         sample_id,
         presence_absence_id,
         present,
-        pa.modified::date as result_ts,
```

Comment:
I think this is a good move overall, but the one thing that sticks out to me is that this changes that column from a date to a time stamp. I worry this could have downstream impacts where code expects a date and doesn't explicitly cast it. One example, EHS view uses this as  test_result_date which it will try to put into a redcap column thats defined as a date. I also don't know if we want that level of specificity for things for return of results?

This could be resolved by casting the result_timestamp to a date instead.

Reply:
> I think this is a good move overall, but the one thing that sticks out to me is that this changes that column from a date to a time stamp. I worry this could have downstream impacts where code expects a date and doesn't explicitly cast it. One example, EHS view uses this as test_result_date which it will try to put into a redcap column thats defined as a date. I also don't know if we want that level of specificity for things for return of results?
> 
> This could be resolved by casting the result_timestamp to a date instead.

Thanks for catching that! When I dropped coalesce in the last push, I accidentally removed the date casting. I'll get that fixed now and change the date cutoff to 2022-06-09.